documentation/user_docs/docs/user_guide/Horace3_to_4.rst

```diff
@@ -69,6 +69,13 @@ Passing a pair of vectors will still be accepted and be internally
 transformed into the correct ``Symop``, however, it is still best to
 switch.
 
+Spaghetti Plot
+--------------
+
+``qwidth`` in ``spaghetti_plot`` arguments now correctly refers to a full-width of the perpendicular
+bins. This means that to achieve the same results in Horace 4 as Horace 3, a user-specified
+``qwidth`` should be twice as large. Default ``qwidths`` have been updated to reflect this change.
```

Comment:
defaults where? For unspecified arguemnts? In documentation? etc

Reply:
Defaults are usually for unspecified arguments, aren't they? I'm not sure what "defaults in documentation" might be and I'm not sure what clarification you want.